src/views/parents/parents.jsx

```diff
@@ -0,0 +1,259 @@
+                <h2>
+                    <FormattedMessage id="parents.overviewTitle" />
+                </h2>
+                <FlexRow className="general-usage">
+                    <div>
+                        <h3><FormattedMessage id="parents.overviewLearningTitle" /></h3>
+                        <p>
+                            <FormattedMessage
+                                id="parents.overviewLearningBody"
+                                values={{
+                                    learningWithScratch: (
+                                        <a href="http://llk.media.mit.edu/scratch/Learning-with-Scratch.pdf">
+                                            <FormattedMessage
+                                                id="parents.learningWithScratchLinkText"
+                                            />
+                                        </a>
+                                    ),
+                                    creativeLearningApproach: (
+                                        <a href="http://www.edutopia.org/kindergarten-creativity-collaboration-lifelong-learning">
+                                            <FormattedMessage
+                                                id="parents.creativeLearningApproachLinkText"
+                                            />
+                                        </a>
+                                    )
+                                }}
+                            />
+                        </p>
+                    </div>
+                    <div>
+                        <h3><FormattedMessage id="parents.overviewCommunityTitle" /></h3>
+                        <p>
+                            <FormattedMessage
+                                id="parents.overviewCommunityBody"
+                                values={{
+                                    communityGuidelines: (
+                                        <a href="/community_guidelines">
+                                            <FormattedMessage
+                                                id="parents.communityGuidelinesLinkText"
+                                            />
+                                        </a>
+                                    ),
+                                    privacyPolicy: (
+                                        <a href="/privacy_policy">
+                                            <FormattedMessage
+                                                id="parents.privacyPolicyLinkText"
+                                            />
+                                        </a>
+                                    )
+                                }}
+                            />
+                        </p>
+                    </div>
+                </FlexRow>
+            </section>
+        </div>
+        <TitleBanner className="faq-banner">
+            <div className="inner">
+                <section id="faq">
```

Comment:
The `FAQ` link in the masthead jumps the page to just below the FAQ header on the page. It may need an additional div with the id `faq` like teacher accounts on the educators page: https://github.com/LLK/scratch-www/blob/6d8235c7e8085e04abc960ea63d9b648d837789b/src/views/teachers/landing/landing.jsx#L131

The developers page also uses `TitleBanner` for the faq, so the css for that page might help as it does jump to the page with the 'FAQ' header showing.